brownie_tokens/forked.py

```diff
@@ -213,6 +213,17 @@ def mint_0x9559Aaa82d9649C7A7b220E7c461d2E74c9a3593(
     token.mint(target, amount, {"from": minter})
 
 
+# Curve LP tokens
+
+
+def mint_0x075b1bb99792c9E1041bA13afEf80C91a1e70fB3(
```

Comment:
This could have some unexpected consequences...  minting LP tokens in a curve pool without also adding assets to the pool will rug the virtual price.  I think better to steal in this case.

Alternatively, could iterate through all the assets within the pool, mint them, and `add_liquidity`

Reply:
Yes, that was a bit hacky. Let's go with stealing!